src/tbb/arena.h

```diff
@@ -397,158 +372,58 @@ class arena: public padded<arena_base>
 
     std::uintptr_t calculate_stealing_threshold();
 
+    unsigned priority_level() { return my_priority_level; }
+
+    int has_request() { return my_total_num_workers_requested; }
+
+    unsigned references() { return my_references.load(std::memory_order_acquire); }
+
+    bool is_arena_workerless() { return my_max_num_workers == 0; }
+
+    bool is_top_priority();
+
+    bool try_join();
+
+    void set_allotment(unsigned allotment);
+
+    std::pair<int, int> update_request(int mandatory_delta, int workers_delta);
```

Comment:
It is unclear what `std::pair<int, int>` semantically means. Consider improve code readability.

Reply:
Added comment